isoutils/exclude

```diff
@@ -0,0 +1,4 @@
+boot/*
```

Comment:
This file can be removed if we switch to listing each exclude on the command line; see comment in [isoutils/isoutils.go](url)

Reply:
Done